### ISSUE TITLE
Test failure in context logger

### ISSUE THREAD

**Issue by phiadaarr** [igrek51/nuclear#9]
I tried to package nuclear (and wat) for nixos and ran into the following problem:

![image](https://github.com/user-attachments/assets/9e945af2-021c-43ee-8c87-20f629f66a06)

If you use nix, you can reproduce the behavior like so:
```sh
git clone --depth=1 -b initPythonWat https://github.com/phiadaarr/nixpkgs
cd nixpkgs
nix-build -A python3Packages.nuclear
```

Any help would be highly appreciated. And btw thanks for your work on wat. This looks really interesting and I cannot wait to use it.

**Comment by igrek51** [comment]
@phiadaarr Looking through the logs I see that there's something wrong with the timezone.
This library prints "Z" suffix at the end of the datetimes in the logs, once it detects that the local timezone is UTC (Zulu timezone). If the timezone is different, is shouldn't be there.
For some reason changing the timezone in the unit test didn't work:
```python
os.environ['TZ'] = 'Europe/Warsaw'
tzset()
```
Yet, I'm not sure why though.

**Comment by phiadaarr** [comment]
Thanks for your quick response! It might be related to the fact the nix builds its packages in sandboxes (see e.g., [here](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#tested-using-sandboxing)). These might imply that the timezone is set to unexpected values. If this problem does not occur on your machine, it is likely to be related to Nix and I will pull it to this community and ask for help there. Let's see.

**Comment by phiadaarr** [comment]
I am afraid the patch did not work:

![image](https://github.com/user-attachments/assets/1c312c26-0380-400a-bd72-2aba36a40b65)

**Comment by igrek51** [comment]
@phiadaarr My bad. I messed up the unit test. Now it should be fixed with https://github.com/igrek51/nuclear/pull/11

**Comment by phiadaarr** [comment]
Awesome, thanks! Now it runs through :) Would it be possible to create a new release that includes this patch? Then I can reference it in nixpkgs.

**Comment by igrek51** [comment]
Yes, it's now released as 2.3.1
